src/components/Room/Room.tsx

```diff
@@ -96,9 +113,24 @@ export function Room({
   }
 
   receiveMessage(message => {
+    const userSettings = settingsContext.getUserSettings()
+    !shellContext.tabHasFocus &&
+      userSettings.playSoundOnNewMessage &&
```

Comment:
`playSoundOnNewMessage` will be `undefined` for returning users since `userSettings` are recalled from Indexed DB when available. Therefore, message audio will not work for those users until they delete their settings (a feature which is now available as of https://github.com/jeremyckahn/chitchatter/commit/e259196942977547ac24b40e069add057c19db57). I think that's okay for now, as it's reflective of more systemic issue with how settings are handled. I can address this as a followup task.

I just wanted to point this out in case anyone finds that audio isn't working for a while once this is merged.

Reply:
we could set the default value of playSoundOnNewMessage to true or false (default value decided) when we get it if its undefined ?

Reply:
Yes. Rather than implementing a one-off fix for this problem, I'd like to have a more systemic solution that gracefully handles new settings properties as they are introduced. I'm wondering if we could achieve this by changing this line: https://github.com/jeremyckahn/chitchatter/blob/e259196942977547ac24b40e069add057c19db57/src/Bootstrap.tsx#L57

To something like this:

```tsx
setUserSettings({ ...userSettings, ...persistedUserSettings })
```

Feel free to give that a try in this PR, or I can address it as a followup task after merging.